lib/flammarion/server.rb

```diff
@@ -67,7 +74,7 @@ def stop
     end
 
     def log(str)
-      # Kernel.puts str
+      Kernel.puts str
```

Comment:
This should probably get commented back out, just in case people are depending on stdout for scripts that use flammarion

```suggestion
      # Kernel.puts str
```